profiling/src/profiling/mod.rs

```diff
@@ -290,6 +300,20 @@ impl TimeCollector {
             }
         }
 
+        #[cfg(feature = "exception_profiling")]
+        if let Some(exception_samples_offset) = exception_samples_offset {
+            let upscaling_info = UpscalingInfo::Proportional {
+                scale: EXCEPTION_PROFILING_INTERVAL.load(Ordering::Relaxed) as f64,
+            };
+            let values_offset: Vec<usize> = vec![exception_samples_offset];
+            match profile.add_upscaling_rule(values_offset.as_slice(), "", "", upscaling_info) {
```

Comment:
No need for a vec here, just use an array, something like:

```suggestion
            let values_offset = [exception_samples_offset];
            match profile.add_upscaling_rule(&values_offset, "", "", upscaling_info) {
```

Probably can do the same to allocation profiling, where this was likely copy/pasted from.